rocketpy/Rocket.py

```diff
@@ -388,48 +384,83 @@ def evaluateThrustToWeight(self):
         self.thrustToWeight.setInputs("Time (s)")
         self.thrustToWeight.setOutputs("Thrust/Weight")
 
-    def evaluateStaticMargin(self):
-        """Calculates and returns the rocket's static margin when
-        loaded with propellant. The static margin is saved and returned
-        in units of rocket diameter or calibers. This function also calculates
-        the rocket center of pressure and total lift coefficients.
+    def evaluateCenterOfPressure(self):
+        """Evaluates rocket center of pressure position relative to user defined
+        rocket reference system. It can be called as many times as needed, as it
+        will update the center of pressure function every time it is called. The
+        code will iterate through all aerodynamic surfaces and consider each of
+        their center of pressure position and derivative of the coefficient of
+        lift as a function of Mach number.
 
         Parameters
         ----------
         None
 
         Returns
         -------
-        self.staticMargin : float
```

Comment:
as `None` was being returned, this documentation is currently wrong at the default branch 

(sadly)